framework/src/SPIDevObj.cpp

```diff
@@ -271,7 +282,7 @@ int SPIDevObj::_modifyReg(uint8_t address, uint8_t clearbits, uint8_t setbits)
 
 int SPIDevObj::bulkRead(DevHandle &h, uint8_t address, uint8_t *out_buffer, int length)
 {
-#if defined(__DF_RPI) || defined(__DF_EDISON)
+#if defined(__DF_RPI) || defined(__DF_EDISON) || defined(__DF_BEBOP)
```

Comment:
@mcharleb This is architecturally horrible - could you look into how we can restructure this to not use ifdefs?